mrjob/spark/mr_spark_harness.py

```diff
@@ -32,6 +32,13 @@ def configure_args(self):
             dest='compression_codec',
             help=('Java class path of a codec to use to compress output.'))
 
+        self.add_passthru_arg(
+            '--passthru-args',
```

Comment:
Could you please rename this to `--job-args`, for consistency with `--job-class`? Having a passthru arg named `passthru_args` is just a bit too recursively confusing.

Also, please specify `dest` explicitly, for consistency with the rest of the `mrjob` library.